docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -67,57 +63,149 @@ The package is published in this location:
+### Encryption Response Content
+
+When encrypting with the `SharingClient`, the SDK returns the following information:
+
+| Property         | Description                                                                                                                                     |
+|:-----------------|:------------------------------------------------------------------------------------------------------------------------------------------------|
+| `status`         | The encryption result status. For a list of possible values and definitions, see [Encryption Response Statuses](#encryption-response-statuses). |
+| `encrypted_data` | The encrypted UID2 token.                                                                                                                       |
+
+### Encryption Response Statuses
+
+| Value                           | Description                                                            |
+|:--------------------------------|:-----------------------------------------------------------------------|
+| `SUCCESS`                       | The raw UID2 was successfully encrypted and a UID2 token was returned. |
+| `NOT_AUTHORIZED_FOR_KEY`        | The requester does not have authorization to use the encryption key.   |
+| `NOT_AUTHORIZED_FOR_MASTER_KEY` | The requester does not have authorization to use the master key.       |
+| `NOT_INITIALIZED`               | The client library is waiting to be initialized.                       |
+| `KEYS_NOT_SYNCED`               | The client has failed to synchronize keys from the UID2 service.       |
+| `ENCRYPTION_FAILURE`            | A generic encryption failure occurred.                                 |
+
+### Decryption Response Content
+
+Whether decrypting with the `BidstreamClient` or the `SharingClient`, the SDK returns the following information:
+
+| Property      | Description                                                                                                                                     |
+|:--------------|:------------------------------------------------------------------------------------------------------------------------------------------------|
+| `status`      | The decryption result status. For a list of possible values and definitions, see [Decryption Response Statuses](#decryption-response-statuses). |
+| `uid`         | The raw UID2 for the corresponding UID2 token.                                                                                                  |
+| `established` | The timestamp indicating when a user first established the UID2 with the publisher.                                                             |
+
+### Decryption Response Statuses
+
+| Value                      | Description                                                             |
```

Comment:
Decryption response codes, and their meanings, are shown in the following table.